### ISSUE TITLE
show Klingon script ({pIqaD}) for each entry 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [De7vID/klingon-assistant#40]
```
There's no official way to write Klingon in Klingon script and the written 
Klingon seen in TV shows, movies, and other media are decoration made up to 
look good and don't generally correspond to anything.

However, there is a semi-official mapping of Okrandian transcription to Klingon 
script characters:
http://www.kli.org/tlh/pIqaD.html
http://klingonska.org/piqad/
```

Original issue reported on code.google.com by `De.vID.jonpIn` on 24 Jan 2013 at 6:17

**Comment by GoogleCodeExporter** [comment]
```
One problem would be how to get a Klingon font to display on Android.
```

Original comment by `De.vID.jonpIn` on 24 Jan 2013 at 6:17

**Comment by GoogleCodeExporter** [comment]
```
Some Klingon fonts are available here:
http://www.evertype.com/fonts/tlh/

However, they don't exactly match the appearance of the typeface shown in the 
TV shows and movies.
```

Original comment by `De.vID.jonpIn` on 24 Jan 2013 at 7:12

**Comment by GoogleCodeExporter** [comment]
```
Font (courtesy of qurgh) and code for handling it in the entry title added in 
revision 7b7ac77b6200.
```

Original comment by `De.vID.jonpIn` on 24 Jan 2013 at 6:23

**Comment by GoogleCodeExporter** [comment]
```
Fixed in revision 2b03f2b824ea.
```

Original comment by `De.vID.jonpIn` on 25 Jan 2013 at 9:02
- Changed state: **Fixed**
